src/transactions/test/LiquidityPoolWithdrawTests.cpp

```diff
@@ -342,5 +343,93 @@ TEST_CASE("liquidity pool withdraw", "[tx][liquiditypool]")
             REQUIRE(acc1.getTrustlineBalance(poolNative1) == 0);
             checkLiquidityPool(*app, poolNative1, 0, 0, 0, 1);
         }
+
+        SECTION("large deposit/withdraw test")
+        {
+            struct Deposit
+            {
+                TestAccount acc;
+                int64_t numPoolShares;
+
+                Deposit(TestAccount const& acc, int64_t numPoolShares)
```

Comment:
I don't think you need this constructor? You should be able to use the default as you're not doing anything special.
I think to insert later on, all you'll need is something like this `deposits.emplace_back(Deposit{acc, size});`

Reply:
Removed